api/etc/file-manager-api.yaml

```diff
@@ -70,12 +70,17 @@
 #       contain a trailing newline.
 #
 # For security reasons, the command is not executed via a subshell. This means
-# all other string interplation/ variables have been disabled. Each argument is
+# all other string interpolation/ variables have been disabled. Each argument is
 # interpreted as a literal string and should not be quoted.
 #
-# Relative paths are expanded from the install directory.
+#>>path<<
```

Comment:
I'm not keen on this mechanism. See my comments on the omnibus builder PR for more details.

Reply:
3dbd297f19e8a0e972c86f24b3a3cfd1d43d1c1a